### PR TITLE
 Add option to visualize label image without sync

### PR DESCRIPTION
**Pull request by wkentaro**
image

The left lower image is what is added by this PR.

### CODE REVIEW

File: jsk_perception/node_scripts/label_image_decomposer.py [hunk 1]
```diff
@@ -120,27 +128,30 @@ def unsubscribe(self):
         self.sub_img.sub.unregister()
         self.sub_label.sub.unregister()
 
-    def _apply(self, img_msg, label_msg):
+    def _apply(self, label_msg, img_msg=None):
```
Comment (furushchev) [inline jsk_perception/node_scripts/label_image_decomposer.py]: @wkentaro I'd ask why you changed the order?

Comment (furushchev) [reply jsk_perception/node_scripts/label_image_decomposer.py]: @wkentaro Oh, img_msg can be None. OK.

File: jsk_perception/node_scripts/label_image_decomposer.py [hunk 2]
```diff
@@ -78,6 +78,7 @@ def __init__(self):
         self.pub_img = self.advertise('~output', Image, queue_size=5)
         self.pub_label_viz = self.advertise('~output/label_viz', Image,
                                             queue_size=5)
+        self._only_label = rospy.get_param('~only_label', False)
```
Comment (furushchev) [inline jsk_perception/node_scripts/label_image_decomposer.py]: @wkentaro I'd recommend to update docs also about this param.

Comment (wkentaro) [reply jsk_perception/node_scripts/label_image_decomposer.py]: Thanks. I updated.

**Pull request merged**